cypress/e2e/top-navigation.cy.ts

```diff
@@ -0,0 +1,55 @@
+/// <reference types='cypress' />
+
+describe("Top navigation test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.get('.navbar a[href="/"]').as("home");
+    cy.get('.navbar a[href="/projects"]').as("projects");
+    cy.get('.navbar a[href="/talks"]').as("talks");
+    cy.get('.navbar a[href="/blog"]').as("blog");
+    cy.get('.navbar a[href="#hire-me"]').as("contact");
+  });
+
+  it("Should check if top navigation buttons have correct addresses", () => {
+    ["Home", "Projects", "Talks", "Blog", "Contact"].forEach((item) => {
+      cy.get(`@${item.toLowerCase()}`).should("include.text", item);
+    });
+  });
+
+  describe("navigation flow", () => {
+    function goBackToHome() {
+      return cy.go("back").location("pathname").should("eq", "/");
+    }
+
+    ["Projects", "Talks", "Blog"].forEach((page) => {
+      it(`Should go to ${page} and back`, () => {
+        cy.get(`@${page.toLowerCase()}`).click();
+        cy.location("pathname").should("eq", `/${page.toLowerCase()}`);
+        cy.get("h2").invoke("text").should("include", page);
+        goBackToHome();
+      });
+    });
+
+    it("Should go to Contact and back", () => {
```

Comment:
This actually doesn't work. Clicking on Contact doesn't move page to the bottom like real browser.
`windows.scrollY=0`